### DISCUSSION TITLE
Smart AutoClicker for Windows?

### DISCUSSION THREAD

**Discussion by Metazolid** [Nain57/Smart-AutoClicker#87] [General]
I've been in need of a program like yours but weirdly enough, those that are available for Windows right now are either borderline sketchy or cost 50€ per license and **all** of them have been made in like 2008, which is apparent by their websites and ancient UI.

And I'm either blind or can't find significant alternatives for those, whereas your App can perfectly fill the gap with its simple, recent interface and accuracy.

If something like this is already in the making, I'd like to betatest it if possible.

as a quick update; I've now opted to use Steam Link to stream my desktop to my phone where I can use AutoClicker. Since Steamlink is optimized for low latency, it's very effective, albeit cumbersome.

**Comment by xTeooo** [comment]
If u wanna spend some time... hit python or even autoit. With openCV u can do everything that this app do or even more.

**Reply by Metazolid** [reply]
I appreciate your suggestion, but I'm not a coder/dev at all. The time spent trying to learn how to code would vastly outweight the benefits my program could bring to the table. Hence me asking if a Windows version was being considered.

**Reply by xTeooo** [reply]
hmm... then maybe try Macrorecorder by Bartels Media ? https://www.macrorecorder.com/

**Reply by Metazolid** [reply]
Just in case someone somehow ends up in here; I found a pretty good and even free desktop alternative called Macro Creator by Pulover. (https://www.macrocreator.com/)
As a heads up; the installer offers an optional Adaware web companion, make sure to decline that and you're good to go.
